site/pricing.html

```diff
@@ -4,6 +4,8 @@
 description: Our flexible pricing options allow companies of any size to have secure authentication, authorization and user management that fits their budget.
 heroHeader: true
 ---
+
+<a name="editions"/>
```

Comment:
Are you using these for deep links on another page? If so, can we use id's instead? 

For example, you'd add `id="editions"` to the `section` (or whatever) element that you wan to allow a link to - and then by adding `#editions` to the end of the URL the link should work. 

Same comment for other uses of these empty `a` tags.

Reply:
Sure, will revise those.